src/plugins/intel_gpu/src/graph/registry/convolution_impls.cpp

```diff
@@ -23,9 +23,7 @@ const std::vector<std::shared_ptr<cldnn::ImplementationManager>>& Registry<convo
         OV_GPU_CREATE_INSTANCE_OCL(ocl::ConvolutionImplementationManager, shape_types::static_shape)
         OV_GPU_CREATE_INSTANCE_OCL(ocl::ConvolutionImplementationManager, shape_types::dynamic_shape,
             [](const cldnn::program_node& node){
-                if (node.can_use(impl_types::onednn))
-                    return false;
-                return node.as<convolution>().use_explicit_padding();
```

Comment:
Which exact impl is expected to be selected after this modification?
Do we have any test cases that cover this scenario?

Reply:
This diff is to choose the agnostic kernel for convolution in compile graph pass. It reduces the compile time in inference.

Checking 'explicit_padding' is WA 2 years ago. Related issue ticket: https://jira.devtools.intel.com/browse/CVS-114379. 
Convolution is supporting 'auto_pad' in IR now.